operator/pkg/helmreconciler/prune.go

```diff
@@ -193,6 +201,10 @@ func (h *HelmReconciler) GetPrunedResources(revision string, includeClusterResou
 		if err != nil {
 			continue
 		}
+		for _, obj := range objects.Items {
+			objName := fmt.Sprintf("%s/%s", obj.GetNamespace(), obj.GetName())
+			metrics.AddResource(objName, gvk.GroupKind())
```

Comment:
since we are not labeling by name, it name needed for the function API?

Reply:
We earlier had over/undercounting problem, especially for shared resources. When we maintain `namespace/name` of resource (grouped by kind), it effectively eliminates that problem. When we report we just report the size of the set. So when we report, accidental addition of a resource already added does not matter.